subscribers/logs/worker/Program.cs

```diff
@@ -19,11 +19,10 @@
 using Npgsql;
 
 namespace Dta.Marketplace.Subscribers.Logger.Worker {
-
     class Program {
         static async Task Main(string[] args) {
             Log.Logger = new LoggerConfiguration()
-                        .MinimumLevel.Debug()
+                        .MinimumLevel.Information()
                         .WriteTo.Console()
```

Comment:
I thought the consensus for the minimum level was '_debug_'. Since writing to console was more for working on the local environment.

Reply:
unfortunately, that made it to the production env aswell...  Will need to add some configuration files at a later point